src/scss/base/_utilities.scss

```diff
@@ -22,4 +22,57 @@ a {
 .main {
   background-color: $white-color;
   margin-top: -3px;
+
+  .Line {
+    width: 100%;
+    height: 0.125rem;
+    border: solid 1px $gary-line;
+    margin: 1.81rem 0 2.313rem;
+  }
+
+  .Line-3 {
+    width: 95%;
+    height: 0.125rem;
+    margin: 2rem auto 0.04rem;
+    border: solid 1px $gary-line;
+  }
+}
+
+.clickable {
+  cursor: pointer;
+}
+
+.flex-center-center {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.Line-1 {
+  content: '';
+  border-color: red;
+  padding: 10px;
+  width: 0.125rem;
+  height: 4.375rem;
+  border: solid 1px $gary-line;
+}
+
+.sm-line {
+  width: 0.125rem;
+  height: 1.5rem;
+  margin: 0.93rem 1.063rem;
+  border: solid 1px $gary-line;
+}
+
+.big-line {
+  width: 0.125rem;
+  height: 4.375rem;
+  margin: 1.125rem 0 0 0;
+  border: solid 1px $gary-line;
+}
+
+// .links li a { color: red; }
+
```

Comment:
- [ ] I suggest that you remove this commented out line of code if its no longer needed. This helps in keeping the code-base clean and readable.